Shared/sdk/CVector4D.h

```diff
@@ -20,28 +20,28 @@
 class CVector4D
 {
 public:
-    CVector4D()
+    constexpr CVector4D() :
+        fX(0),
+        fY(0),
+        fZ(0),
+        fW(0)
```

Comment:
this can just be `int fX = 0;` in member declaration

Reply:
actually, you're right. I just wasnt sure if I should do it that way.